.github/workflows/ci.yaml

```diff
@@ -36,27 +36,27 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: conda-incubator/setup-miniconda@v2
+      # More info on options: https://github.com/conda-incubator/setup-miniconda
+      - name: Setup Micromamba
+        uses: mamba-org/provision-with-micromamba@main
         with:
-            auto-update-conda: true
-            use-mamba: true
-            python-version: ${{ matrix.python-version }}
-            miniforge-variant: Mambaforge
-            environment-file: environment.yml
-            activate-environment: openfe
+          environment-file: environment.yml
+          environment-name: openfe
+          extra-specs: |
+            python==${{ matrix.python-version }}
 
       - name: "Install"
-        run: python setup.py develop --no-deps
+        run: python -m pip install --no-deps -e .
 
       - name: "Test imports"
         run: |
           # if we add more to this, consider changing to for + env vars
-          python -Ic "import openfe"
+          python -Ic "import openfe; print(openfe.__version__)"
```

Comment:
@mikemhenry @richardjgowers just adding this extra bit here to have a quick version sanity check.